ml-package-versions.yml

```diff
@@ -2,7 +2,8 @@ sklearn:
   package_info:
     pip_release: "scikit-learn"
     install_dev: |
-      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
+      # This installs scikit-learn from the default branch
```

Comment:
@smurching Added a comment here to clarify this command install scikit-learn from the default branch.